rasa/nlu/tokenizers/whitespace_tokenizer.py

```diff
@@ -11,15 +11,29 @@ class WhitespaceTokenizer(Tokenizer, Component):
 
     provides = ["tokens"]
 
+    defaults = {
+        # text will be tokenized as case sensitive by default
+        "case_sensitive": True
+    }
+
+    def __init__(self, component_config: Dict[Text, Any] = None) -> None:
+        """Construct a new tokenizer using the WhitespaceTokenizer framework."""
+
+        super(WhitespaceTokenizer, self).__init__(component_config)
+
     def train(
         self, training_data: TrainingData, config: RasaNLUModelConfig, **kwargs: Any
     ) -> None:
-
+        case_sensitive = self.component_config["case_sensitive"]
```

Comment:
this should go to `tokenize` method

Reply:
component_config is not assessable inside tokenize method which is why i added the logic in train method. If i change the signature of tokenize, there are many methods using it right?

Reply:
fixed the PR comments